doc/api/test.md

```diff
@@ -134,6 +134,43 @@ Any subtests that are still outstanding when their parent finishes
 are cancelled and treated as failures. Any subtest failures cause the parent
 test to fail.
 
+## Retrying tests
+
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+Failed tests can be retried via a hardcoded `{ retries }` option, or
+via the dynamic `retries()` TestContext prototype method.
+
+Via the `.currentAttempt` property, the current attempt (the number of times
```

Comment:
Can you remove the leading `.` from the property names.

Also, `currentAttempt` is a bit confusing IMO. I would expect it to be one-based and not only specific to retries. For example, if I run a test with no retries, it will run once. I would expect `currentAttempt` to be one on that single run.